app/templates/content/cookies.html

```diff
@@ -97,7 +97,50 @@ <h3>Session cookies</h3>
             </tr>
             </tbody>
         </table>
-  
+
+        <h3>Measuring website usage</h3>
+
+        <p class="govuk-body">
+            We use Google Analytics to measure how you use the Digital Marketplace so we can improve it based on user needs.
+        </p>
+
+        <p class="govuk-body">Google Analytics sets cookies that store anonymised information about:
+
+        <ul class="govuk-list govuk-list--bullet">
+            <li>how you got to Digital Marketplace</li>
+            <li>the pages you visit on Digital Marketplace and government digital services, and how long you spend on each page</li>
+            <li>what you click on while you&rsquo;re using the service</li>
+        </ul>
+        </p>
+
+        <p class="govuk-body">
+            We do not allow Google to use or share the data about how you use this site.
+        </p>
+
+        <table class="content-table">
+            <thead>
+            <tr>
+                <th>Name</th>
+                <th>Purpose</th>
+                <th>Expires</th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr>
+                <td>_ga</td>
+                <td>Helps us count how many people visit Digital Marketplace by tracking if you’ve visited the service before
+                </td>
+                <td>2 years</td>
+            </tr>
+            <tr>
+                <td>_gid</td>
+                <td>Helps us count how many people visit Digital Marketplace by tracking if you’ve visited the service before
+                </td>
+                <td>24 hours</td>
+            </tr>
+            </tbody>
+        </table>
+
```

Comment:
...but this indentation doesn't match the "standard" you reformatted the other file to (?)